id/mod.go

```diff
@@ -1,7 +1,60 @@
 package id
 
+import (
+	"crypto/sha256"
+	"go.dedis.ch/dela/mino"
+	"math/big"
+)
+
 type Id interface {
-	GetLength() byte
+	GetLength() int
 	GetBase() byte
 	GetDigit(pos int) byte
-}
\ No newline at end of file
+}
+
+type ArrayId struct {
+	Id   []byte
```

Comment:
nit, `ID` here

Reply:
Left it as a private member for now. Do you think it makes sense to make it public?